### PR TITLE
route graded submission deep links to assignment details so user can see a grade

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-13272
affects: Student
release note: none

Test plan
------------
https://instructure.atlassian.net/browse/MBL-13272

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Core/CoreTests/Extensions/URLComponentsExtensionsTests.swift [hunk 1]
```diff
@@ -55,4 +55,14 @@ class URLComponentsExtensionsTests: XCTestCase {
...
     }
+
+    func testOriginIsNotification() {
+        var url = URLComponents.parse("https://foobar.com/courses/165/assignments/900/submissions/1")
+        XCTAssertFalse(url.originIsNotification)
+
+        url.originIsNotification = true
+        XCTAssertTrue(url.originIsNotification)
+
+        XCTAssertEqual(url.url?.absoluteString, "https://foobar.com/courses/165/assignments/900/submissions/1?origin=notification")
```
Comment (armstrongnate) [inline Core/CoreTests/Extensions/URLComponentsExtensionsTests.swift]: Setting `originIsNotification = false` isn't exercised. Want to add a test for that?

Comment (bluwave) [reply Core/CoreTests/Extensions/URLComponentsExtensionsTests.swift]: good catch

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-13272](https://instructure.atlassian.net/browse/MBL-13272)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.23% | 93.24% | -0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1009567&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by armstrongnate** [comment]
This is working fine. Looks like submission comment notifications have the same url as graded submissions so I've asked Peyton if it's okay that submission comments go to assignment details now.

**Pull request merged**